lib/swim/join-sender.js

```diff
@@ -113,11 +112,6 @@ function JoinCluster(opts) {
     this.maxJoinDuration = numOrDefault(opts.maxJoinDuration,
         MAX_JOIN_DURATION);
 
-    // We want to abort the joining process if we have failed
```

Comment:
I moved this param into `config.js` so that I can make the integration tests "faster" and also because all configs should eventually belong in `config.js`